shipit.yml

```diff
@@ -1,4 +1,4 @@
 ci:
   hide:
-    -  buildkite/kubernetes-deploy-gem-nightly
+    -  buildkite/krane-gem-nightly
```

Comment:
Should we kill this? I'm not convinced we're getting value out of it.

Reply:
I feel we don't have a lot of changes happening daily that justifies it, so 👍 for 🔥 it.